Source/WebCore/html/canvas/WebGLRenderingContextBase.h

```diff
@@ -440,7 +440,7 @@ class WebGLRenderingContextBase : public GraphicsContextGL::Client, public GPUBa
 
     void prepareForDisplayWithPaint() final;
     void paintRenderingResultsToCanvas() final;
-    RefPtr<PixelBuffer> paintRenderingResultsToPixelBuffer();
+    RefPtr<PixelBuffer> paintRenderingResultsToPixelBuffer(bool flipY);
```

Comment:
Sorry to be a stickler, but we tend to use something like:

```
enum class FlipY : uint8_t {
  NotFlipped,
  Flipped
}
```

Reply:
So that the bool parameter makes more sense at the call sites.

Reply:
> So that the bool parameter makes more sense at the call sites.

"and I would have gotten away with it too if it weren't for you meddling kids"